src/dev/build/tasks/bundle_fleet_packages.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Task, read, downloadToDisk } from '../lib';
+
+const EPR_URL = 'https://epr-snapshot.elastic.co';
+
+interface FleetPackage {
+  name: string;
+  version: string;
+  checksum: string;
+}
+
+export const BundleFleetPackages: Task = {
+  description: 'Bundling fleet packages',
+
+  async run(config, log, build) {
+    log.info('Fetching fleet packages from package registry');
+    log.indent(4);
+
+    const configFilePath = config.resolveFromRepo('fleet_packages.json');
+    const fleetPackages = (await read(configFilePath)) || '[]';
+
+    await Promise.all(
+      JSON.parse(fleetPackages).map(async (fleetPackage: FleetPackage) => {
+        const archivePath = `${fleetPackage.name}-${fleetPackage.version}.zip`;
+        const archiveUrl = `${EPR_URL}/epr/${fleetPackage.name}/${fleetPackage.name}-${fleetPackage.version}.zip`;
+
+        const destination = config.resolveFromRepo(
+          'x-pack/plugins/fleet/server/bundled_packages',
```

Comment:
We could probably move this out of the Fleet plugin directory to a higher-level location if desired. Putting it here just was the lowest level of effort required to having the existing logic function.

Reply:
I think you need to use `build.resolvePath()` instead of `config.resolveFromRepo`. When I tested running a build, the packages were not copied to the build directory (`./build`) so they weren't included in the final build.

Reply:
I think we'd need to implement that separately from a build step for dev. I'd like guidance from @elastic/kibana-operations about the recommended way to add this to bootstrap for development.

We probably should also make sure that the directory used for dev doesn't get copied into the build so that builds are clean. This may already be the case, but for example, if I manually put a zip in `x-pack/plugins/fleet/server/bundled_packages` in the repo, we should verify these don't get copied into the build before the BundleFleetPackages step is executed. I thought maybe checking `src/dev/build/tasks/copy_source_task.ts` would be good enough, but I actually don't see where any of the `x-pack` files are copied in?